challenges/2021_cloud_base_height/02_explore_transformed_data.ipynb

```diff
@@ -0,0 +1,1532 @@
+{
```

Comment:
<p>I'm not sure what this plot means? Maybe some more explanation, or possibly updating the plot to be more intuitive? Maybe you could do it two part - zero vs non zero and then a pie chart of everything else?</p><p>Also I thinks its good to put compute and display in separate cells, so I suggest moving the plotting code to a separate cell from the compute code here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>